### PR TITLE
SLAS-808 Allow user to link self custodial eth

### PR DESCRIPTION
**Pull request by workingmonk**
## Refs

- **Issue**: SLAS-808

## What?

Allow users to link self-custodial wallet in addition to the custodial wallet

### CODE REVIEW

**Comment by workingmonk** [comment]
Screen Shot 2022-12-19 at 4 50 09 AM

File: src/core/ui/components/drop-down/inner.tsx [hunk 1]
```diff
@@ -163,6 +163,22 @@ export const Inner = ({ context }: Props) => {
             </span>
           </div>
         )}
+        {authSettings.availableWeb3LoginMethods.length > 0 &&
+          user.wallet &&
+          user.rawWallet.walletTypeMap &&
+          user.rawWallet.walletTypeMap[user.wallet] === 'managed_wallet' && (
```
Comment (nedrocks) [inline src/core/ui/components/drop-down/inner.tsx]: Can we make this an enum or constant please?

**Review by nedrocks — APPROVED** [review]

**Comment by workingmonk** [comment]
Screen Shot 2022-12-19 at 4 44 34 PM

**Comment by workingmonk** [comment]
`+ Connect Personal Wallet` looks good. Let me know if you want to revert to `+ Connect Wallet`

**Review by nedrocks — COMMENTED** [review]
This looks good but it's missing the popover info button still

**Comment by nedrocks** [comment]
Can you please indent the `connect personal wallet` underneath the wallet header so it's lined up with the wallet address

Can you please decrease the padding between the wallet address and connect personal wallet.

Can you please add the info popover button for the custodial wallet

**Comment by workingmonk** [comment]
Screen Shot 2022-12-20 at 12 53 31 PM

**Review by nedrocks — APPROVED** [review]

**Comment by workingmonk** [comment]
once user attaches their wallet, this is the view
image

**Pull request merged**